package.json

```diff
@@ -109,6 +109,7 @@
     "css-loader": "^5.2.6",
     "cssom": "^0.4.4",
     "cypress": "^7.4.0",
+    "cypress-failed-log": "^2.9.2",
```

Comment:
Just checking this out while the PR is in draft and we're looking at the 500 issues. If it seems helpful I'll look to add it in its own PR, rather than here